superchain/configs/mainnet/soneium.toml

```diff
@@ -0,0 +1,61 @@
+name = "Soneium"
+chain_id = 1868
+public_rpc = "https://rpc.soneium.org"
+sequencer_rpc = ""
```

Comment:
Can you just update this field to be the same as the public_rpc? I think you can just plug it into the `.env` file and run `just add-chain`